packages/honeycomb-opentelemetry-web/src/web-vitals-autoinstrumentation.ts

```diff
@@ -481,6 +482,25 @@ export class WebVitalsInstrumentation extends InstrumentationAbstract {
       [`${attrPrefix}.resource_load_time`]: resourceLoadDuration,
     });
 
+    lcpEntry?.element?.getAttributeNames().forEach((attrName) => {
+      // Skip non data-* attributes
+      if (!attrName.startsWith('data-')) {
```

Comment:
I think it would be safer / more correct to use the [`dataset` API](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes#javascript_access) in JavaScript rather than iterating over all the attributes if it's available!

Reply:
OH. Heck yeah!